metadata-ingestion/src/datahub/ingestion/source/dremio/dremio_source.py

```diff
@@ -281,6 +275,15 @@ def get_workunits_internal(self) -> Iterable[MetadataWorkUnit]:
         """
 
         self.source_map = self._build_source_map()
+
+        space_urn = self.dremio_aspects.get_container_space_urn()
+        container = DremioContainer("Spaces", "", [], self.dremio_catalog.dremio_api)
+        yield from self.dremio_aspects.populate_container_mcp(space_urn, container)
+
+        source_urn = self.dremio_aspects.get_container_source_urn()
+        container = DremioContainer("Sources", "", [], self.dremio_catalog.dremio_api)
+        yield from self.dremio_aspects.populate_container_mcp(source_urn, container)

```

Comment:
Same - we should not require this now that browse path entry is using raw strings.